From 6ef69a26126ee4e69a25392fd456b8a66c51dffd Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Tue, 15 Nov 2016 02:46:55 +0000
Subject: [PATCH] Fix errors found by clang

Fixes errors like

../../git/src/hash.cpp:282:19: error: ordered comparison between pointer and zero ('const unsigned char *' and 'int')
            if(fdht->base>0){
               ~~~~~~~~~~^~

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 src/hash.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/hash.cpp b/src/hash.cpp
index 42161579f..52f419b0c 100644
--- a/src/hash.cpp
+++ b/src/hash.cpp
@@ -279,7 +279,7 @@ void file_data_hasher_t::hash()
 		MAP_FILE|
 #endif
 		MAP_SHARED,fd,0);
-	    if(fdht->base>0){		
+	    if(fdht->base != (void *) -1){
 		/* mmap is successful, so set the bounds.
 		 * if it is not successful, we default to reading the fd
 		 */
